pilot/pkg/proxy/envoy/v2/monitoring.go

```diff
@@ -114,7 +130,9 @@ var (
 		"pilot_proxy_convergence_time",
 		"Delay between config change and all proxies converging.",
 		[]float64{1, 3, 5, 10, 20, 30, 50, 100},
+		monitoring.WithLabels(errTag),
```

Comment:
not sure we need this now. What do you think?

Reply:
Unfortunately this must be defined if we still want `proxiesConvergeDelayCdsErrors`, `proxiesConvergeDelayEdsErrors`, `proxiesConvergeDelayRdsErrors` and `proxiesConvergeDelayLdsErrors`. Otherwise any of above "error" metric reporting will be counted as `proxiesConvergeDelay` because `errTag` is not defined.

Let's take an example. Assume we have such a scenario that CDS, LDS, EDS pushed without error but RDS failed. And theoretically we will have:
```
pilot_proxy_convergence_time_count{err="cds"} = 0
pilot_proxy_convergence_time_count{err="eds"} = 0
pilot_proxy_convergence_time_count{err="lds"} = 0
pilot_proxy_convergence_time_count{err="rds"} = 1
# this is 1 because we report it after all xDS pushed is done
pilot_proxy_convergence_time_count{err=""} = 1
```
but in reality we will get:
```
# this is 2 because the err tag is not defined and the rds error reporting is counted together here
pilot_proxy_convergence_time_count{""} = 2
```

Reply:
yeah I mean I am not sure we really need the proxiesConvergeDelayRdsErrors metrics at all. Since we already have it for the individual XDS. It's a very weird metric to measure "the time we started a push until it got an rds error". What's more useful, that you are adding now, is " the time to do an rds push" and maybe "the time to do an rds error"

Reply:
Got it. I agree `proxiesConvergeDelayXdsErrors` metric series can be removed. Let's use `pilot_proxy_convergence_time_count` simply for the total convergence time.